mmdet/models/necks/fpn.py

```diff
@@ -34,6 +35,14 @@ class FPN(nn.Module):
         norm_cfg (dict): Config dict for normalization layer. Default: None.
         act_cfg (str): Config dict for activation layer in ConvModule.
             Default: None.
+        upsample_cfg (dict): Config dict for interpolate layer.
+            Default: `dict(mode='nearest')`
+        extra_convs_source (str, optional): The source feature map of the extra
```

Comment:
`extra_convs_source` have some overlaps with `extra_convs_on_inputs`. But to keep compatible with previous configs, I have left the key `extra_convs_on_inputs` intact.

Reply:
We can deprecate this option and remove it in future releases.

Reply:
We may use a single argument `add_extra_convs` to specific behaviors.
- False
- `'on_input'`
- `'on_lateral'`
- `'on_output'`